src/modules/aws.rs

```diff
@@ -0,0 +1,31 @@
+use std::env;
+
+use ansi_term::Color;
+
+use super::{Context, Module};
+
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    if let Ok(aws_profile) = env::var("AWS_PROFILE") {
```

Comment:
It might be neater to drop the if statement and use `let aws_profile = env::var("AWS_PROFILE").ok?;`